### PR TITLE
BZ1947814:VPA Operator uninstall doesn't document manual cleanups

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1947814

Preview:  [Uninstalling the Vertical Pod Autoscaler Operator]( https://deploy-preview-41349--osdocs.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-vertical-autoscaler.html#nodes-pods-vertical-autoscaler-uninstall_nodes-pods-vertical-autoscaler)

### CODE REVIEW

**Comment by mburke5678** [comment]
@palonsoro PTAL

**Comment by mburke5678** [comment]
@weinliu PTAL

**Comment by palonsoro** [comment]
LGTM

File: modules/nodes-pods-vertical-autoscaler-uninstall.adoc [hunk 1]
```diff
@@ -27,3 +29,105 @@ You can remove a specific VPA using the `oc delete vpa <vpa-name>` command. The
 . Optional: To remove all operands associated with the Operator, in the dialog box, select *Delete all operand instances for this operator* checkbox.
 
 . Click *Uninstall*.
+
+. Optional: Use the {product-title} CLI to remove the VPA components:
```
Comment (bobfuru) [inline modules/nodes-pods-vertical-autoscaler-uninstall.adoc:33]: I've generally seen us avoiding the attribute and writing as "OpenShift CLI", such as here: https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html

File: modules/nodes-pods-vertical-autoscaler-uninstall.adoc [hunk 2]
```diff
@@ -27,3 +29,105 @@ You can remove a specific VPA using the `oc delete vpa <vpa-name>` command. The
...
+[source,terminal]
+----
+$ oc delete mutatingwebhookconfigurations/vpa-webhook-config
+----
+
+.. List any VPA custom resources: 
++
+[source,terminal]
+----
+$ oc get  verticalpodautoscalercheckpoints.autoscaling.k8s.io,verticalpodautoscalercontrollers.autoscaling.openshift.io,verticalpodautoscalers.autoscaling.k8s.io -o wide --all-namespaces
```
Comment (bobfuru) [inline modules/nodes-pods-vertical-autoscaler-uninstall.adoc:46]: There is an extra space that renders in the output after `$oc get`

File: modules/nodes-pods-vertical-autoscaler-uninstall.adoc [hunk 3]
```diff
@@ -27,3 +29,105 @@ You can remove a specific VPA using the `oc delete vpa <vpa-name>` command. The
...
+----
+$ oc delete verticalpodautoscalercontroller.autoscaling.openshift.io/default -n openshift-vertical-pod-autoscaler
+----
++
+[source,terminal]
+----
+$ oc delete verticalpodautoscaler.autoscaling.k8s.io/vpa-recommender -n my-project
+----
+
+.. List any VPA custom resource definitions (CRD):
```
Comment (bobfuru) [inline modules/nodes-pods-vertical-autoscaler-uninstall.adoc:79]:
```suggestion
.. List any VPA custom resource definitions (CRDs):
```

**Review by bobfuru — COMMENTED** [review]
Looks good! Just a few minor comments.

**Pull request merged**